ui/src/dashboards/components/CellEditorOverlay.tsx

```diff
@@ -24,18 +23,45 @@ import {STATIC_LEGEND} from 'src/dashboards/constants/cellEditor'
 // Types
 import * as QueriesModels from 'src/types/queries'
 import * as SourcesModels from 'src/types/sources'
-import {NotificationAction, TimeRange} from 'src/types'
+import {NotificationAction, TimeRange, CellType} from 'src/types'
 import {Template} from 'src/types/tempVars'
 import {
   Cell,
   Legend,
   CellQuery,
   NewDefaultCell,
   QueryType,
+  DecimalPlaces,
+  FieldOption,
+  ThresholdType,
+  TableOptions,
+  NoteVisibility,
+  Axes,
 } from 'src/types/dashboards'
 import {Links, ScriptStatus} from 'src/types/flux'
+import {ColorString, ColorNumber} from 'src/types/colors'
+
+interface ConnectedProps {
+  queryDrafts: CellQuery[]
```

Comment:
It looks like `timeRange`, `onUpdateQueryDrafts`, and `onChangeScript` aren't used anywhere and don't need to be connected.